cmd/oras/repository/tags.go

```diff
@@ -97,3 +97,17 @@ func isDigestTag(tag string) bool {
 	_, err := digest.Parse(dgst)
 	return err == nil
 }
+
```

Comment:
This `NewRemoteTarget` would not live here, just here to keep the changes simple.